minifi/minifi-c2/minifi-c2-jetty/src/main/java/org/apache/nifi/minifi/c2/jetty/JettyServer.java

```diff
@@ -100,7 +113,42 @@ public static void main(String[] args) throws Exception {
         server.join();
     }
 
-    private static WebAppContext loadWar(final File warFile, final String contextPath, final ClassLoader parentClassLoader) throws IOException {
+    private static SSLContext buildSSLContext(C2Properties properties) {
+        KeyStore keyStore;
+        KeyStore truststore;
+
+        File keyStoreFile = Paths.get(C2_SERVER_HOME).resolve(properties.getProperty(MINIFI_C2_SERVER_KEYSTORE)).toFile();
+        logger.debug("keystore path: " + keyStoreFile.getPath());
```

Comment:
Placeholders should be used instead of string concatenation:
```suggestion
        logger.debug("Loading Key Store [{}]", keyStoreFile.getPath());
```